utils/hare-bootstrap

```diff
@@ -281,7 +286,12 @@ fi
 abort_if_RC_leader_election_is_impossible
 
 read _ join_ip <<< $(get_server_nodes | grep -w $(node-name))
-read _ join_peers <<< $(get_server_nodes | grep -vw $(node-name))
+
+# get_server_nodes will produce multiple lines. We'll join them with space.
+# Note: in case when get_server_nodes is empty list, consequent awk and tr will exit with non-zero code.
+# In order to swallow this error, `|| true` is needed.
+
+join_peers="$(get_server_nodes | grep -vw "$(node-name)" | awk '{print $2}' | tr '\n' ' ' || true)"
```

Comment:
Consider `xargs` - it might be a shorter and simple alternative, for example:
```
$ cat | xargs echo
aa
bb
cc
aa bb cc
```

Reply:
As far as I understand, `xargs` will replace last `tr` only. If so, I don't see much of simplification.